package.json

```diff
@@ -137,15 +140,14 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-tsdoc": "^0.2.16",
     "eslint-plugin-unicorn": "^42.0.0",
-    "eslint-plugin-vue": "^8.6.0",
+    "eslint-plugin-vue": "^9.0.0",
     "eslint-plugin-vuejs-accessibility": "^1.1.1",
     "husky": "^7.0.1",
     "jest": "^26.6.3",
     "jest-transform-stub": "^2.0.0",
     "lint-staged": "^12.3.7",
     "module-alias": "^2.2.2",
     "npm-run-all": "^4.1.5",
-    "pinia": "^2.0.11",
```

Comment:
We apparently had `pinia` both in dependencies and devDependencies, with somewhat different versions specified.